src/resolver.ts

```diff
@@ -105,13 +123,21 @@ export class DisputeResolver {
   @confirmResolve
   async accept(disputeID: string, execute = false): Promise<void> {
     log.info(`Accepting dispute ${disputeID}...`)
-    await this.commit(disputeID, DisputeResolution.Accept, execute)
+    if (await this.validateStatuteOfLimitations(disputeID)) {
```

Comment:
This is good but I can see us refactoring it to include more conditions later, what do you think? 

At a later point we can have a validateCharter() that does the limitations one, double jeopardy, etc. within the function, print the error and return. If all good then commits.

Reply:
I agree but I didn't want to over-engineer it until we see the need.